src/MissionManager/SurveyMissionItem.cc

```diff
@@ -1062,7 +1062,7 @@ bool SurveyMissionItem::_appendMissionItemsWorker(QList<MissionItem*>& items, QO
         MissionItem* item = new MissionItem(seqNum++,
                                             MAV_CMD_DO_SET_CAM_TRIGG_DIST,
                                             MAV_FRAME_MISSION,
-                                            0.0,                    // trigger distance (off)
+                                            -1.0,                   // trigger distance (off)
```

Comment:
Yes - with 0, for cameras with power control (on PX4) it turns the camera off (lens closes, camera powers down). We don't want that to happen during the turnarounds. We want the camera to be powered, but not taking photos.